### PR TITLE
Updated header styles

### PR DESCRIPTION
**Pull request by Lukewh**
- Element heights consistent
- Colours consistent
- Spacing improvements
- Small mobile styling update

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7702/

File: jujugui/static/gui/src/app/assets/css/_header.scss [hunk 1]
```diff
@@ -87,11 +90,11 @@
     display: inline-block;
     border-radius: 2px;
     border-right-width: 0;
-    border: 1px solid $light-mid-grey;
+    border: 1px solid $color-mid-light;
     transition: background-color .2s;
     background-color: $canvas-background;
-    padding: 13px 11px;
-    margin: 12px 0 11px 12px;
+    padding: 11px 11px;
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/assets/css/_header.scss]: Could just be `padding: 11px;`

File: jujugui/static/gui/src/app/components/header-search/_header-search.scss [hunk 2]
```diff
@@ -60,26 +60,29 @@
 
   &__input {
     font-weight: 300;
-    padding: 10px 10px 10px 40px;
+    padding: 8px 40px 8px 40px;
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/header-search/_header-search.scss]: Could be `padding: 8px 40px`

File: jujugui/static/gui/src/app/components/model-actions/_model-actions.scss [hunk 3]
```diff
@@ -20,8 +18,8 @@
   &__export,
   &__import,
   &__share {
-    padding: 10px 11px;
-    border: 1px solid $light-mid-grey;
+    padding: 11px 11px;
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/model-actions/_model-actions.scss]: Could be `padding: 11px;`

**Comment by Lukewh** [comment]
Thanks @hatched. Thanks @huwshimi - good catches. My mind blanked out 'optimisations' while tweaking things by pixels here and there....

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7707/

**Comment by Lukewh** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**